### PR TITLE
ReleaseNotes: Updated changelog and release notes for 7.5.4

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+### Features and enhancements
+
+* **AzureMonitor**: Add support for Microsoft.AppConfiguration/configurationStores namespace. [#32123](https://github.com/grafana/grafana/pull/32123), [@deesejohn](https://github.com/deesejohn)
+* **TablePanel**: Make sorting case-insensitive. [#32435](https://github.com/grafana/grafana/pull/32435), [@kaydelaney](https://github.com/kaydelaney)
+
+### Bug fixes
+
+* **AzureMonitor**: Add support for Virtual WAN namespaces. [#32935](https://github.com/grafana/grafana/pull/32935), [@joshhunt](https://github.com/joshhunt)
+* **Bugfix**: Make sure we have proper padding when scrolling is added to bar gauge. [#32411](https://github.com/grafana/grafana/pull/32411), [@mckn](https://github.com/mckn)
+* **Datasource**: prevent default datasource named "default" from causing infinite loop. [#32949](https://github.com/grafana/grafana/pull/32949), [@jackw](https://github.com/jackw)
```
Comment (osg-grafana) [inline CHANGELOG.md:15]:
```suggestion
* **Datasource**: Prevent default data source named "default" from causing infinite loop. [#32949](https://github.com/grafana/grafana/pull/32949), [@jackw](https://github.com/jackw)
```

Comment (osg-grafana) [reply CHANGELOG.md:15]: I just changed this upstream too.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+* **Prometheus**: Allow exemplars endpoint in data source proxy. [#32802](https://github.com/grafana/grafana/pull/32802), [@zoltanbedi](https://github.com/zoltanbedi)
```
Comment (osg-grafana) [inline CHANGELOG.md:16]:
```suggestion
* **Prometheus**: Allow query_exemplars endpoint in data source proxy. [#32802](https://github.com/grafana/grafana/pull/32802), [@zoltanbedi](https://github.com/zoltanbedi)
```
Seems like this endpoint name needs to be updated. Is this correction okay? If so, I can change this upstream.

**Review by osg-grafana — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+
+### Features and enhancements
+
+* **AzureMonitor**: Add support for Microsoft.AppConfiguration/configurationStores namespace. [#32123](https://github.com/grafana/grafana/pull/32123), [@deesejohn](https://github.com/deesejohn)
+* **TablePanel**: Make sorting case-insensitive. [#32435](https://github.com/grafana/grafana/pull/32435), [@kaydelaney](https://github.com/kaydelaney)
+
+### Bug fixes
+
+* **AzureMonitor**: Add support for Virtual WAN namespaces. [#32935](https://github.com/grafana/grafana/pull/32935), [@joshhunt](https://github.com/joshhunt)
+* **Bugfix**: Make sure we have proper padding when scrolling is added to bar gauge. [#32411](https://github.com/grafana/grafana/pull/32411), [@mckn](https://github.com/mckn)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:14]:
```suggestion
* **Bugfix**: Add proper padding when scrolling is added to bar gauge. [#32411](https://github.com/grafana/grafana/pull/32411), [@mckn](https://github.com/mckn)
```

File: docs/sources/release-notes/release-notes-7-5-4.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+* **TablePanel:** Make sorting case-insensitive. [#32435](https://github.com/grafana/grafana/pull/32435), [@kaydelaney](https://github.com/kaydelaney)
+
+### Bug fixes
+
+* **AzureMonitor:** Add support for Virtual WAN namespaces. [#32935](https://github.com/grafana/grafana/pull/32935), [@joshhunt](https://github.com/joshhunt)
+* **Bugfix:** Make sure we have proper padding when scrolling is added to bar gauge. [#32411](https://github.com/grafana/grafana/pull/32411), [@mckn](https://github.com/mckn)
+* **Datasource:** Prevent default datasource named "default" from causing infinite loop. [#32949](https://github.com/grafana/grafana/pull/32949), [@jackw](https://github.com/jackw)
+* **Prometheus:** Allow exemplars endpoint in data source proxy. [#32802](https://github.com/grafana/grafana/pull/32802), [@zoltanbedi](https://github.com/zoltanbedi)
+* **Table:** Fix table data links, so they refer to correct row after sorting. [#32571](https://github.com/grafana/grafana/pull/32571), [@torkelo](https://github.com/torkelo)
```
Comment (osg-grafana) [inline docs/sources/release-notes/release-notes-7-5-4.md:22]:
```suggestion
* **AzureMonitor**: Add support for Virtual WAN namespaces. [#32935](https://github.com/grafana/grafana/pull/32935), [@joshhunt](https://github.com/joshhunt)
* **Bugfix**: Make sure we have proper padding when scrolling is added to bar gauge. [#32411](https://github.com/grafana/grafana/pull/32411), [@mckn](https://github.com/mckn)
* **Datasource**: Prevent default data source named "default" from causing infinite loop. [#32949](https://github.com/grafana/grafana/pull/32949), [@jackw](https://github.com/jackw)
* **Prometheus**: Allow query_exemplars endpoint in data source proxy. [#32802](https://github.com/grafana/grafana/pull/32802), [@zoltanbedi](https://github.com/zoltanbedi)
* **Table**: Fix table data links so they refer to correct row after sorting. [#32571](https://github.com/grafana/grafana/pull/32571), [@torkelo](https://github.com/torkelo)
```

**Review by osg-grafana — APPROVED** [review]

**Review by achatterjee-grafana — APPROVED** [review]
Suggested a minor change, otherwise LGTM

File: docs/sources/release-notes/release-notes-7-5-4.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+### Features and enhancements
+
+* **AzureMonitor:** Add support for Microsoft.AppConfiguration/configurationStores namespace. [#32123](https://github.com/grafana/grafana/pull/32123), [@deesejohn](https://github.com/deesejohn)
+* **TablePanel:** Make sorting case-insensitive. [#32435](https://github.com/grafana/grafana/pull/32435), [@kaydelaney](https://github.com/kaydelaney)
+
+### Bug fixes
+
+* **AzureMonitor**: Add support for Virtual WAN namespaces. [#32935](https://github.com/grafana/grafana/pull/32935), [@joshhunt](https://github.com/joshhunt)
+* **Bugfix**: Make sure we have proper padding when scrolling is added to bar gauge. [#32411](https://github.com/grafana/grafana/pull/32411), [@mckn](https://github.com/mckn)
```
Comment (osg-grafana) [inline docs/sources/release-notes/release-notes-7-5-4.md:19]:
```suggestion
* **Bugfix**: Add proper padding when scrolling is added to bar gauge. [#32411](https://github.com/grafana/grafana/pull/32411), [@mckn](https://github.com/mckn)
```

File: docs/sources/release-notes/release-notes-7-5-4.md (hunk 4 above)
Comment (achatterjee-grafana) [reply docs/sources/release-notes/release-notes-7-5-4.md]: The colon should be bold.

**Comment by osg-grafana** [comment]
Commit changes per Andrej Ocenas via internal Slack.

**Comment by aocenas** [comment]
@osg-grafana I guess based on this https://github.com/grafana/grafana-github-actions/issues/21, this change https://github.com/grafana/grafana/pull/32973#discussion_r613211006 should be reverted?

**Comment by osg-grafana** [comment]
> @osg-grafana I guess based on this [grafana/grafana-github-actions#21](https://github.com/grafana/grafana-github-actions/issues/21), this change [#32973 (comment)](https://github.com/grafana/grafana/pull/32973#discussion_r613211006) should be reverted?

@oddlittlebird, can you take this PR over and resolve this issue. I am feeling a time crunch atm.

**Comment by aocenas** [comment]
I am going to merge this as it is right now. I think if there is a styling issue we can fix it later.

**Pull request merged**